inventory/src/ts/components/inventory-window.tsx

```diff
@@ -45,6 +46,16 @@ export class InventoryWindow extends React.Component<InventoryWindowProps, Inven
       client.EquipItem(group.getFirstItemID());
     }
   }
+  
+  simDblClick(callOnDblClick: (...args : any[]) => void, ...args : any[]) {
+	  let now:number = Date.now();
+	  if (now - this.lastClicked < 400) {
+      callOnDblClick.apply(this, args);
```

Comment:
Just preference again... I prefer using the spread operator to just call the function normally rather than using apply.  This does however only work with TypeScript right now if you have the callback set to any.

```
simDblClick(onDblClick: any, ...dblClickArgs: any[]) {
  // check for clicks...
  if ( double clicked)  {
    onDblClick(...dblClickArgs);
  }
}
```